include/functions.h

```diff
@@ -1347,14 +1347,14 @@ void func_800AA178(u32);
 View* View_New(GraphicsContext* gfxCtx);
 void View_Free(View* view);
 void View_Init(View*, GraphicsContext*);
-void func_800AA358(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up);
-void func_800AA3F0(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up);
+void View_LookAt(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up);
+void View_LookAtInternal(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up);
```

Comment:
I really don't want to give this function such an innocuous name. It's unused in the code, and probably no-one should use it, since the code really expects that dirty flag to get set.

Reply:
Maybe `Unsafe` or `Unused` if you don't like `Internal`?

Reply:
I like `Internal` over `Unused`, calling something "unused" is to me the absolute last resort 😅 
`Unsafe` could definitely make sense though and would still communicate well that it's not really supposed to be used under any normal circumstances.

Reply:
Whatever it's called, it sounds like it shouldn't be in functions.h any more anyway.

Reply:
unsafe means more to me than internal does, if that helps with deciding

Reply:
Well, if it's not used externally, and we think it shouldn't be used externally, that suggests that not making it available externally is appropriate.

Reply:
dunno, it seems to me like it was an alternate version of the call that existed to be used externally, but they ended up just using the safe version for everything

Reply:
I wasn't sure if we ever took things out of functions.h so I left it in; would probably favour taking it out.

I really don't think anyone has a reason to call this function, as it probably won't do what they expect and there is a working version right next to it.

People seem to like `Unsafe`, so, renamed: e1eb2b03f

Reply:
Yeah, that makes sense 👍

Reply:
functions.h will be broken up eventually, though: it's horrible to open that file in VSCode with a small amount of ram because of its size and cross-references. And it's just bad organisationally to have such a huge header anyway.

Reply:
whether or not if this particular function was meant to be internal or external is certainly debatable, cause of how little info we have. But I would personally lean on the side of intended to be external, cause  i dont see them making an internal helper function for this one thing. But its a gut feeling thing, dont really have any evidence

Reply:
ofc yeah, it will be broken up when each system has its own header. but for now you can just view functions.h as the declarations for externally available functions